admin/class-admin.php

```diff
@@ -18,23 +16,22 @@ class Admin {
 	 * Class constructor.
 	 */
 	public function __construct() {
-		$this->init();
-		$this->init_ajax();
 	}
 
 	/**
 	 * Initialize.
 	 *
 	 * @return void
 	 */
-	private function init() {
-		$admin_notices = new \EDAC\Admin\Admin_Notices();
+	public function init() {
+		
+		$admin_notices = new Admin_Notices();
 		$admin_notices->init_hooks();
 
-		$widgets = new \EDAC\Admin\Widgets();
+		$widgets = new Widgets();
 		$widgets->init_hooks();
 
-		new EDAC\REST_Api();
```

Comment:
API needs to be reachable outside of Admin, so moved REST_Api init to class-plugin.php.